### ISSUE TITLE
k8petstore doesn't support non-default namespaces.

### ISSUE THREAD

**Issue by jayunit100** [kubernetes/kubernetes#13160]
Looks like the `nodeport` upgrade broke namespaces in k8petstore.

 #8145 fixes this.  If someone can do a quick review, it also adds a smoke test.

However, I'm creating this as a separate issue (in case #8145 is blocked, I will create an individual PR here for fixing k8petstore-nodeport.sh )

**Comment by jayunit100** [comment]
please close after #8145 is merged.  If that doesn't happen soon ill create a patch specific for this.

**Comment by dims** [comment]
@jayunit100 can this be closed now?

**Comment by jayunit100** [comment]
Yup !
